extensions/JSTestAdapter.md

```diff
@@ -0,0 +1,143 @@
+## Seamlessly running JavaScript tests with VSTest
```

Comment:
There is already a folder inside docs called extensions, consider moving there if it makes sense to you.